setupUBN.sh

```diff
@@ -0,0 +1,27 @@
+# This setup script can be used to install all the dependencies required to clone and run the project on Ubuntu machines
+
+!/bin/bash
+
+Run the go_installer script with the parent directory as an argument
+./go_installer.sh ../
+echo "Installed go" 
+go version
+sudo apt update -y
+sudo apt install openssh-server -y
+sudo apt install net-tools -y
+sudo apt install git -y
+sudo apt install gcc -y
+sudo apt install libfuse-dev -y
+sudo apt install fuse -y
+sudo apt install fuse3 -y
+sudo apt install libfuse3-dev -y
+echo "Installed all dependencies" -y
+
+# Open the file /etc/fuse.conf and uncomment the line user_allow_other
```

Comment:
You can also add how to install Blobfuse2 which includes [configuring](https://learn.microsoft.com/en-us/linux/packages) the apt repository.

Reply:
yes we shall also add step here to enlist microsoft linux repo for ubuntu and may be install blobfuse2 latest version as part of this as well. This will help you setup syslog filters so that logs can be directed to syslog in a separate file.

Reply:
Also, if we can add steps to install secpack that would be great.